src/rules/utils.ts

```diff
@@ -0,0 +1,685 @@
+import {
+  AST_NODE_TYPES,
+  ESLintUtils,
+  TSESLint,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import { basename } from 'path';
+import { version } from '../../package.json';
+
+const REPO_URL = 'https://github.com/jest-community/eslint-plugin-jest';
+
+export const createRule = ESLintUtils.RuleCreator(name => {
+  const ruleName = basename(name, '.ts');
+  return `${REPO_URL}/blob/v${version}/docs/rules/${ruleName}.md`;
+});
+
+/**
+ * A `Literal` with a `value` of type `string`.
+ */
+export interface StringLiteral extends TSESTree.Literal {
+  value: string;
+}
+
+type StringNode = StringLiteral | TSESTree.TemplateLiteral;
+
+/**
+ * Checks if the given `node` is a {@link StringLiteral}.
+ *
+ * @param {Node} node
+ *
+ * @return {node is StringLiteral}
+ */
+export const isStringLiteral = (node: TSESTree.Node): node is StringLiteral =>
```

Comment:
don't export this, it's not used. Mind going over all exports and just keeping the ones actually used outside of this file as `export`s?

Reply:
Yup that's at the top of my list of things to do during the week - same w/ checking all the generics :)